python/paddle/fluid/parallel_executor.py

```diff
@@ -46,6 +46,10 @@ def __init__(self,
                 improve performance in some cases, defalut False.
             share_vars_from(ParallelExecutor, default None): If provied,
                 it will share variables from the specified ParallelExecutor.
+            use_default_grad_scale(bool, default True): If set True, a default
+                scale value equal to `1./device_count` would be multiplied to
+                the gradients. Otherwise, a customized scale value should be
```

Comment:
to gradients of each device? and then aggregated?

Reply:
Thanks, followed the comment.